pkg/dbmate/db_test.go

```diff
@@ -86,7 +86,7 @@ func TestDumpSchema(t *testing.T) {
 	require.NoError(t, err)
 
 	// create and migrate
-	err = db.CreateAndMigrate()
+	err = db.CreateAndMigrate(false)
```

Comment:
In general I'm not a fan of boolean args like this (see https://www.martinfowler.com/bliki/FlagArgument.html), since it's not obvious to the caller what the arg represents. Could we split this into separate methods?

Reply:
Good point. Will do.

Reply:
done.